<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>267</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>315</width>
    <height>267</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>315</width>
    <height>267</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UART Timing Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>uartIcon.jpg</normaloff>uartIcon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="13" column="0">
     <widget class="QLabel" name="label_Bytes">
      <property name="text">
       <string>Bytes:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_Parity">
      <property name="text">
       <string>Parity :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="options_Parity">
      <item>
       <property name="text">
        <string>NONE</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ODD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EVEN</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="input_ByteCount"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_DataBits">
      <property name="text">
       <string>Data Bits:</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QLabel" name="result_TimePerPacket">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="options_DataBits">
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Baudrate">
      <property name="text">
       <string>Baudrate(bps):</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="options_StopBits">
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="input_Baudrate"/>
    </item>
    <item row="15" column="1">
     <widget class="QLabel" name="result_BitsPerByte">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QLabel" name="result_TimePerByte">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_StopBits">
      <property name="text">
       <string>Stop Bits:</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_TimePerPacket">
      <property name="text">
       <string>Time per Packet:</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_TimePerByte">
      <property name="text">
       <string>Time per Byte:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_BitPerByte">
      <property name="text">
       <string>Bits per Byte:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="Line" name="splitter_right">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="Line" name="splitter_left">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
